doc/userguide/configuration/suricata-yaml.rst

```diff
@@ -2610,3 +2610,20 @@ detect thread. For each output script, a single state is used. Keep in
 mind that a rule reload temporary doubles the states requirement.
 
 .. _deprecation policy: https://suricata.io/about/deprecation-policy/
+
+.. _suricata-yaml-config-hardening:
+
+Configuration hardening
+-----------------------
+
+The `security` section of suricata.yaml is meant to provide in-depth security configuration options.
+
+For now, only one setting is available `limit-noproc`: a boolean to prevent process creation by Suricata.
+If you do not need Suricata to create other processes (for LUA scripts for instance), this will
```

Comment:
suggestion:
- [...] this will
+ [...] enable this to
```

Reply:
ok